.github/workflows/build.yaml

```diff
@@ -0,0 +1,116 @@
+name: "build"
```

Comment:
The first version of this build has three steps:

1. coding standards
2. dependency analysis
3. static analysis